src/ui/components/DevTools.tsx

```diff
@@ -53,6 +54,9 @@ function DevTools({
   const { recording, isAuthorized, loading: recordingQueryLoading } = hooks.useGetRecording(
     recordingId
   );
+  if (recording?.workspace) {
+    mixpanel.register({ workspaceName: recording.workspace.name });
```

Comment:
This is probably not what we want because it will associate this workspace with this user until the next time they view a recording, which is not accurate.